sdk/textanalytics/ai-text-analytics/test/textAnalyticsClient.spec.ts

```diff
@@ -105,10 +107,116 @@ describe("[AAD] TextAnalyticsClient", function() {
         })
       );
       const allInputs = enInputs.concat(esInputs);
-
       const results = await client.analyzeSentiment(allInputs);
       assert.equal(results.length, testDataEn.length + testDataEs.length);
       assertAllSuccess(results);
+      results.map((result) =>
+        (result as AnalyzeSentimentSuccessResult).sentences.map((sentence) =>
+          assert.isUndefined(sentence.minedOpinions)
+        )
+      );
+    });
```

Comment:
this addresses your comment here https://github.com/Azure/azure-sdk-for-js/pull/10484#discussion_r467245297.

Reply:
thank you!